client/my-sites/post-type-list/index.jsx

```diff
@@ -132,8 +126,13 @@ class PostTypeList extends Component {
 		return Math.min( pageCount, 5 );
 	}
 
-	postIdsFromPosts( posts ) {
-		return ! isEmpty( posts ) ? posts.map( post => post.ID ) : [];
+	updateRecentPostViews() {
+		const { posts, siteId } = this.props;
+
+		if ( posts && posts.length > 0 ) {
+			const postIds = posts.map( post => post.ID );
+			requestRecentPostViews( siteId, postIds );
+		}
```

Comment:
because `httpData` maintains its own freshness there shouldn't be too much need for this optimization in the view code. did you find that it was too slow?